SOURCES/consul-template.service

```diff
@@ -0,0 +1,16 @@
+[Unit]
+Description=consul-template Watches a series of templates on the file system, writing new changes when Consul is updated.
+Documentation=http://www.consul-template.io
+After=network-online.target
```

Comment:
Consul-template is not dependent on consul-server. The two are not
necessarily installed on the same system. In my use case I will have a
separate consul cluster set up which all my consul-template clients will
use.

Does that make sense?

On Monday, July 4, 2016, dan phrawzty <notifications@github.com> wrote:

> In SOURCES/consul-template.service
> <https://github.com/tomhillable/consul-rpm/pull/37#discussion_r69428996>:
>
> > @@ -0,0 +1,16 @@
> > +[Unit]
> > +Description=consul-template Watches a series of templates on the file system, writing new changes when Consul is updated.
> > +Documentation=http://www.consul-template.io
> > +After=network-online.target
>
> Would it make sense to require the consul service itself, first?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/tomhillable/consul-rpm/pull/37/files/980394ffffe40c3c1072f404eed71e66a6ccb1d3#r69428996>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/ABCKu552gevotYoiiwD8H0_ZpomeE6VSks5qSM2QgaJpZM4JCr7n>
> .
>